sql/core/src/main/scala/org/apache/spark/sql/execution/command/commands.scala

```diff
@@ -237,15 +237,18 @@ case class ExplainCommand(
     logicalPlan: LogicalPlan,
     override val output: Seq[Attribute] =
       Seq(AttributeReference("plan", StringType, nullable = true)()),
-    extended: Boolean = false)
+    extended: Boolean = false,
+    codegen: Boolean = false)
```

Comment:
As you know, I just implemented this because SPARK-14251 describes `explain codegen select * ...`.
I thought SPARK-14251 aims to reduce the debugging efforts by showing the full info in a single command.
If that is not a goal, we should change it. 

You are the issue reporter of SPARK-14251. :)
If you want to do this in another way, I can change this right now. This is the very right time. :)

Could you update [SPARK-14251](https://issues.apache.org/jira/browse/SPARK-14251) for me before making `CODEGEN` command?

Thank you, @rxin .